pyproject.toml

```
@@ -1,12 +1,12 @@
 [project]
 name = "pagerduty-mcp"
-version = "0.2.1"
+version = "1.0.0-alpha"
 description = "PagerDuty's official local MCP (Model Context Protocol) server which provides tools to interact with your PagerDuty account directly from your MCP-enabled client."
 readme = "README.md"
 requires-python = ">=3.12,<3.13"
 dependencies = [
     "mcp[cli]>=1.8.0",
-    "pagerduty>=2.0.0",
+    "pagerduty>=5.3.0",
```

```suggestion
    "pagerduty~=5.3",
```

for compatibility with https://github.com/PagerDuty/pagerduty-mcp-server/pull/59